packages/maskbook/src/plugins/ITO/apis/index.ts

```diff
@@ -153,7 +153,7 @@ export async function getAllPoolsAsSeller(address: string) {
         body: stringify({
             query: `
             {
-                sellInfos (where: { seller: "${address.toLowerCase()}" }) {
+                sellInfos (where: { seller: "${address.toLowerCase()}" } first: 10000) {
```

Comment:
If my recall is correct, Thegraph only returns the first 1000 records.

Reply:
Well, 1000 is actually enough.